examples/nlp/text_classification_tf_keras/README.md

```diff
@@ -0,0 +1,37 @@
+# Tensorflow Multi-class Text Classification Example
+
+This example demonstrates how to integrate and train a Tensorflow model on Determined's API. 
+We will implement a multiclass classifier that will be trained to perform sentiment 
+analysis on a dataset of Amazon product reviews, classifying reviews into 5 categories (0-5 stars). 
+It is adapted from Tensorflow's
+[word embedding](https://www.tensorflow.org/tutorials/text/word_embeddings) tutorial.
+
+## Files
+* `model_def.py`: main trial class for your model containing the model definition, initialization
```

Comment:
model_def.py: The core code for training. This includes building and compiling the model.